tools/run_azure_travis_integration_tests.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 source tools/base_travis_integration_tests.sh
 
+install_pycloudlib() {
+    git clone https://github.com/canonical/pycloudlib.git
+    cd pycloudlib
+    pip install -r requirements.txt
+    pip install -e .
```

Comment:
I think we can drop this now because pycloudlib's requirements.txt does just that